contracts/implementation/PoolSwapLibrary.sol

```diff
@@ -417,16 +418,18 @@ library PoolSwapLibrary {
      */
     function getUpdatedAggregateBalance(UpdateData calldata data)
         external
-        pure
+        view
         returns (
             uint256 _newLongTokens,
             uint256 _newShortTokens,
+            uint256 _addedLongBalance,
```

Comment:
From my understanding this is the amount that is extracted from the burn as a burning fee. addedLongBalance sounds vague why not _longBurnFee and _shortBurnFee

Reply:
Yep good idea. I suck at variable naming